code/modules/shuttle/shuttle_events/meteors.dm

```diff
@@ -2,8 +2,8 @@
 	spawning_list = list(/obj/effect/meteor)
 
 /datum/shuttle_event/simple_spawner/meteor/post_spawn(atom/movable/spawnee)
-	ADD_TRAIT(spawnee, TRAIT_FREE_HYPERSPACE_MOVEMENT, src)
-	..()
+	. = ..()
```

Comment:
We don’t listen to a return value, so I’d prefer it if you kept it pure to avoid confusion
```suggestion
	..()
```

Reply:
Generally speaking I (we) (royal) will always save the return value of parent unless there's an explicit reason not to, for the case that one may be added, to prevent future bugs

Reply:
Also the only other implementation of this *did* keep parent call and you wrote it so like idk 

![image](https://github.com/tgstation/tgstation/assets/51863163/1256f963-a9da-471e-9ba0-bd423a860fb7)

Reply:
Epic I’m gonna drown myself very cool